housing_modeling/src/test.py

```diff
@@ -0,0 +1,66 @@
+import os
+from pathlib import Path
+
+import joblib
+import numpy as np
+import pandas as pd
+from sklearn.metrics import mean_squared_error
+
+from utils import create_logger
+
+path = Path(os.getcwd())
+
+LOGGING_PATH = os.path.join(path.parent.parent, "housing_modeling/logs")
+HOUSING_PATH = os.path.join(path.parent.parent, "datasets/housing")
+MODEL_PATH = os.path.join(path.parent.parent, "datasets/modeling")
```

Comment:
the paths are repeated in multiple modules.. perhaps move them to a common module ?